### PR TITLE
Remove nc from descriptions of OPS files

### PR DESCRIPTION
**Pull request by dwhswenson**
Files can also be .db. Change recommended as an aside in https://github.com/openpathsampling/mini-tutorials/issues/5#issuecomment-911562562.

### CODE REVIEW

**Comment by dwhswenson** [comment]
This is ready for review and comment. I will leave it open for at least 24 hours, merging no earlier than Sat 04 Sep 04:00 GMT (00:00 my local).

File: paths_cli/commands/contents.py [hunk 1]
```diff
@@ -22,16 +22,16 @@
 
 @click.command(
     'contents',
-    short_help="list named objects from an OPS .nc file",
+    short_help="list named objects from an OPS output file",
```
Comment (sroet) [inline paths_cli/commands/contents.py:25]: wouldn't this more suitable to be `storage` file, or just `file`?
```suggestion
    short_help="list named objects from an OPS storage file",
```
```suggestion
    short_help="list named objects from an OPS file",
```

File: paths_cli/commands/contents.py [hunk 2]
```diff
@@ -22,16 +22,16 @@
...
 )
 @INPUT_FILE.clicked(required=True)
 @click.option('--table', type=str, required=False,
               help="table to show results from")
 def contents(input_file, table):
-    """List the names of named objects in an OPS .nc file.
+    """List the names of named objects in an OPS output file.
```
Comment (sroet) [inline paths_cli/commands/contents.py:31]: wouldn't this more suitable to be `storage` file, or just `file`?
```suggestion
    """List the names of named objects in an OPS storage file.
```

```suggestion
    """List the names of named objects in an OPS file.
```

**Review by sroet — COMMENTED** [review]
I am a bit unsure about the wording (it could also be used on an "input" file), but feel free to ignore. This LGTM otherwise

**Comment by dwhswenson** [comment]
Thanks -- I like the suggestion of `storage file`. I had the same concerns, and had considered `database file` (but that's too close the to the `.db` suffix, making it sound SimStore-specific) and just `file` (but with the YAML/JSON/TOML input files that will be possible after #43, there's more than one kind of "OPS file" -- and the YAML/JSON/TOML files are definitely better thought of as "input," so don't want to use the word "input file"!)

"Storage file" finds the right balance. Going with that.

**Pull request merged**